research/src/pages/popup/popup.research.explainer.mdx

```diff
@@ -155,27 +203,138 @@ function supportsPopup() {
+
+- `delegatesfocus`. When present on the popup element itself, this causes the **first focusable descendent** of the popup to be focused when the popup is shown. (Eventually, this attribute might be made applicable to any element, not just popups, to control focus behavior more generally.)
+
+If both `autofocus` and `delegatesfocus` are both present on the popup element, the popup element is focused when shown.
+
+
+## Anchoring
+
+A new attribute, `anchor`, can be used on a popup element to refer to the popup's "anchor". The value of the attribute is a string idref corresponding to the `id` of another element (the "anchor element"). This anchoring relationship is used for two things:
+
+1. Establish the provided anchor element as an [“ancestor”](#nearest-open-ancestral-popup) of this popup, for light-dismiss behaviors. In other words, the `anchor` attribute can be used to form nested popups.
+2. The referenced anchor element could be used by the [Anchor Positioning feature](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/main/CSSAnchoredPositioning/explainer.md).
+
+
+## Backdrop
+
+Akin to modal `<dialog>` and fullscreen elements, popups allow access to a `::backdrop` pseudo element, which is a full-screen element placed directly behind the popup in the top layer. This allows the developer to do things like blur out the background when a popup is showing:
+
+```html
+<div popup=popup>I'm a popup</div>
+<style>
+[popup]::backdrop {
+  backdrop-filter: blur(3px);
+}
+</style>
+```
+
+Note that in contrast to the `::backdrop` pseudo element for modal dialogs and fullscreen elements, the `::backdrop` for a popup is styled by a UA stylesheet rule `pointer-events: none !important`, which means it cannot trap clicks outside the popup. This ensures the "click outside" light dismiss behavior continues to function.
+
+# Behaviors
+
+
+
+## Automatic Dismiss Behavior
+
+For elements that are displayed on the top layer via this API, there are a number of things that can cause the element to be removed from the top-layer, besides the ones [described above](#showing-and-hiding-a-popup). These fall into three main categories:
+
+* **One at a Time.** Another element being added to the top-layer causes existing top layer elements to be removed. This is typically used for “one at a time” type elements: when one popup is shown, other popups should be hidden, so that only one is on-screen at a time. This is also used when “more important” top layer elements get added. For example, fullscreen elements should close all open popups.
+* **Light Dismiss**. User action such as clicking outside the element, hitting Escape, or causing keyboard focus to leave the element should all cause a showing popup to be hidden. This is typically called “light dismiss”, and is discussed in more detail in [this section](#light-dismiss).
```

Comment:
just a consistency nit, putting the period inside the bold like the previous

```suggestion
* **Light Dismiss.** User action such as clicking outside the element, hitting Escape, or causing keyboard focus to leave the element should all cause a displayed popup to be hidden. This is typically called “light dismiss”, and is discussed in more detail in [this section](#light-dismiss).
```

Reply:
Thanks! I hate when those aren't consistent.

On the `displayed` vs `showing`, I fought with that throughout. I wanted to avoid "open"/"closed" because we're using "show"/"hide", but it's awkward in many cases. I like "displayed", I might try to use that more.